### PR TITLE
[Experimental] Fix test failures caused by adjusting path of artifacts and the target framework of templates in built SDK

### PR DESCRIPTION
**Pull request by GangWang01**
Fix test failures caused by adjusting path of artifacts and the target framework of templates in built SDK.

### CODE REVIEW

File: src/Tests/Microsoft.NET.Build.Containers.IntegrationTests/ProjectInitializer.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@ public sealed class ProjectInitializer
 
     static ProjectInitializer()
     {
-        var relativePath = Path.Combine("..", "packaging", "build", "Microsoft.NET.Build.Containers.targets");
+        var relativePath = Path.Combine("..", "..", "Containers", "packaging", "build", "Microsoft.NET.Build.Containers.targets");
```
Comment (GangWang01) [inline src/Tests/Microsoft.NET.Build.Containers.IntegrationTests/ProjectInitializer.cs:17]: Need to put into artifacts temp folder.

File: src/Containers/containerize/containerize.csproj [hunk 2]
```diff
@@ -17,4 +17,11 @@
     <ProjectReference Include="..\Microsoft.NET.Build.Containers\Microsoft.NET.Build.Containers.csproj" />
   </ItemGroup>
 
+  <!-- Copy to TestExecutionDirectory for using in integration tests. -->
+  <Target Name="CopyDllToTestExecutionDirectory" AfterTargets="Build">
```
Comment (GangWang01) [inline src/Containers/containerize/containerize.csproj:21]: Need to copy self-contained build output to artifact temp folder and then copy to Helix. 
Due to my poor knowledge on msbuild targets, this didn't work. Need to improve.

File: src/Tests/UnitTests.proj [hunk 3]
```diff
@@ -61,6 +61,8 @@
 
       <Testpackages Include="$(ArtifactsTmpDir)testpackages\**\*.*"/>
 
+      <!-- include Container artifacts for running test in Helix -->
```
Comment (GangWang01) [inline src/Tests/UnitTests.proj:64]: Pending to implement. Refer to how to deal with `<Testpackages Include="$(ArtifactsTmpDir)testpackages\**\*.*"/>`.

File: src/Tests/Microsoft.NET.Build.Containers.IntegrationTests/EndToEndTests.cs [hunk 4]
```diff
@@ -153,10 +155,9 @@ public async Task EndToEnd_NoAPI()
 
         newProjectDir.Create();
         privateNuGetAssets.Create();
-        var repoGlobalJson = Path.Combine("..", "..", "..", "..", "global.json");
-        File.Copy(repoGlobalJson, Path.Combine(newProjectDir.FullName, "global.json"));
```
Comment (vlada-shubina) [inline src/Tests/Microsoft.NET.Build.Containers.IntegrationTests/EndToEndTests.cs]: I think this was needed. We need to troubleshoot it.

File: src/Containers/containerize/containerize.csproj [hunk 5]
```diff
@@ -17,4 +17,11 @@
...
+    <ItemGroup>
+      <ContainerizeFiles Include="$(ArtifactsBinDir)containerize\$(Configuration)$(SdkTargetFramework)\**\*.*" />
```
Comment (vlada-shubina) [inline src/Containers/containerize/containerize.csproj:23]:
```suggestion
      <ContainerizeFiles Include="$(ArtifactsBinDir)containerize\$(Configuration)\$(SdkTargetFramework)\**\*.*" />
```

File: src/Tests/Microsoft.NET.Build.Containers.IntegrationTests/EndToEndTests.cs (hunk 4 above)
Comment (GangWang01) [reply src/Tests/Microsoft.NET.Build.Containers.IntegrationTests/EndToEndTests.cs]: This global.json specifies SDK version. For SDK test framework, it only has the built SDK and its version changes over time. So it's not needed in SDK test. Does it make sense?

**Pull request closed without merging**